client/app/styles/onboarding.styl

```diff
@@ -72,6 +91,42 @@ header
 #confirmation-figure .svg-wrapper
         background-color  #2ce096
 
+// agreeemnt step elements
+.agreements
+    display         flex
+    flex-direction  row
+    flex-wrap       wrap
+    width           150%
+    margin          1.5em 0em 1.5em -25%
+
+    .agreements-element
+        flex-grow       1
+        padding         0em 1.5%
+        margin          1em 0em
+        flex-basis      30%
+
+        p
+            margin      auto
+            width       90%
+
+    #home .svg-wrapper
```

Comment:
No need to change for this time, but I think we have to keep in mind to avoid referring to ids in CSS.